x/evm/types/precompiles.go

```diff
@@ -0,0 +1,28 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+
+package types
+
+const (
+	P256PrecompileAddress         = "0x0000000000000000000000000000000000000100"
+	Bech32PrecompileAddress       = "0x0000000000000000000000000000000000000400"
+	StakingPrecompileAddress      = "0x0000000000000000000000000000000000000800"
+	DistributionPrecompileAddress = "0x0000000000000000000000000000000000000801"
+	ICS20PrecompileAddress        = "0x0000000000000000000000000000000000000802"
+	VestingPrecompileAddress      = "0x0000000000000000000000000000000000000803"
+	BankPrecompileAddress         = "0x0000000000000000000000000000000000000804"
+)
+
+// AvailableStaticEVMExtensions defines the full list of all available EVM extension addresses.
+//
+// NOTE: To be explicit, this list does not include the dynamically registered EVM extensions
+// like the ERC-20 extensions.
+var AvailableStaticEVMExtensions = []string{
```

Comment:
nit: I would deprecate the EVM Extensions name in favour of only Static/dynamic precompile to avoid confusion.

```suggestion
var AvailableStaticPrecompiles = []string{
```